src/main/java/edu/harvard/iq/dataverse/DatasetPage.java

```diff
@@ -5477,4 +5477,15 @@ public boolean isFileDeleted (DataFile dataFile) {
 
         return dataFile.getDeleted();
     }
+
+    /**
+     * Add Signposting
+     * @return String
+     */
+    public String getSignpostingLinkHeader() {
+        if (!workingVersion.isReleased())
+            return "DRAFT";
```

Comment:
why DRAFT? is this expected with SignPosting? Or should the header not be included if the dataset is draft?